client/landing/gutenboarding/onboarding-block/design-selector/style.scss

```diff
@@ -196,6 +197,7 @@ $design-selector-selection-space: 285px;
 	position: absolute;
 	top: 0;
 	width: 100%;
+	cursor: pointer;
```

Comment:
This is a bit nitpicky, but we should consider moving this rule to the parent `.page-layout-selector__item`.  This parent has about 2 pixels of padding around the edge, and the hover effect for the border seems to be triggered here.  This gives a few pixels around there edge where the hover effect is triggered before the cursor-pointer is.

![nitpicky-selector](https://user-images.githubusercontent.com/28742426/72761071-d1492e00-3ba8-11ea-98d9-4308d79a05f4.gif)

A shot of the parent __item's padding:
![Screen Shot 2020-01-20 at 5 12 16 PM](https://user-images.githubusercontent.com/28742426/72761115-f8076480-3ba8-11ea-95f6-bf8eca9d5213.png)

Reply:
> […] we should consider moving this rule to the parent `.page-layout-selector__item`.

👍 Good spot, let's make sure this is applied to the clickable element.

Reply:
Good catch! I have made the relevant changes so that the pointer cursor also shows when hovering at the border. See: cdddaaf.